HeartRateMonitor.py

```diff
@@ -0,0 +1,25 @@
+class HRM:
+
+    def __init__(self, times=[], voltages=[], time_units='s', t1 = None, t2 = None):
+        self.times = times
+        self.voltages = voltages
+        self.time_units = time_units
+        self.averagehr = -1
+        if t1 is None:
+            t1 = self.times[0]
+        if t2 is None:
+            t2 = self.times[len(self.times)-1]
+        self.t1 = t1
+        self.t2 = t2
+
+    def avghr(self):
+        from AvgHR import avghr
+        self.averagehr = avghr(self.times, self.voltages, self.t1, self.t2)
```

Comment:
also, be sure to consistently use snake_case to name variables and functions throughout your code. `averagehr` should be `average_hr` canonically. :)